### PR TITLE
Visit log pagination issues

### PR DESCRIPTION
**Pull request by SwathiSelvakumaran**
last item on the visit log list not being shown fully

image

progress text not shown on some forum questions

image

### CODE REVIEW

**Comment by phillipsk** [comment]
I created a new emulator `small phone` and attached a video of the screens which cannot scroll and views are cut off with a small screen size

image


https://github.com/user-attachments/assets/61db190e-2f50-4f14-9f16-490f9c6c1327

File: app/src/main/res/layout/fragment_visit_form3.xml [hunk 1]
```diff
@@ -1,270 +1,277 @@
...
-        android:gravity="center"
-        android:padding="5dp"
-        android:text="@string/visit_log"
-        android:textColor="@color/black"
-        android:textSize="22sp"
-        android:textStyle="bold"
-        app:fontFamily="Poppins"/>
-    <androidx.cardview.widget.CardView
-        android:id="@+id/cv_visitlog"
+    android:layout_height="match_parent">
```
Comment (phillipsk) [inline app/src/main/res/layout/fragment_visit_form3.xml:4]: nit: fillViewport="true"

**Review by phillipsk — CHANGES_REQUESTED** [review]
Many of the visit log fragment layouts need the ability to scroll. The most effective way would be to edit each file as you have done with fragment_visit_form3 and fragment_visit_form4

Another consideration would be some sort of abstraction with a new layout file and using an include block as below but this might overcomplicate the task and require 6-7 more layout files. 

<?xml version="1.0" encoding="utf-8"?>
<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:fillViewport="true">

    <include
        layout="@layout/fragment_visit_form1" />

</androidx.core.widget.NestedScrollView>

**Comment by SwathiSelvakumaran** [comment]
Hey Kevin,
Which method would you like to incorporate for all the visit log fragment layouts and do you know which all layout must be incorporated for the changes.

**Comment by phillipsk** [comment]
You could implement the same approach you have already done with form 3-4
change the all the layouts to be scroll view.

**Review by phillipsk — APPROVED** [review]
If you want to enhance the performance in a separate PR you could look into replacing the nested LinearLayouts with Constraint layout

https://developer.android.com/develop/ui/views/layout/improving-layouts/optimizing-layouts 

This PR is good to merge as is without this enhancement, that can be reviewed in another task. There are outstanding merge conflicts with fragment_profile.xml

Resolve those and merge this in

**Pull request merged**